wasmtime/_func.py

```diff
@@ -201,20 +209,22 @@ def invoke(idx, params_ptr, results_ptr, params):  # type: ignore
 
     try:
         for i in range(0, len(param_tys)):
-            params.append(extract_val(Val(params_ptr[i])))
+            params.append(Val(params_ptr[i]).value)
```

Comment:
I think this may not be quite right because we're not given ownership of parameters?